public/index.html

```diff
@@ -1,19 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
     <!-- Use Internet Explorer Edge Standards mode -->
-    <meta http-equiv="x-ua-compatible" content="IE=edge" >
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="theme-color" content="#000000">
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <meta http-equiv="x-ua-compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta name="theme-color" content="#000000" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <title>Washington Opioid Clinical Assessment</title>
+    <script type="text/javascript">
+      var _paq = window._paq || [];
+      /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+      _paq.push(["trackPageView"]);
+      _paq.push(["enableLinkTracking"]);
```

Comment:
You dropped this: `  _paq.push(["setUserId", "REPLACE THIS WITH THE USER ID, IF KNOWN']);` 
Don't we know that in fEMR?

Reply:
@mcjustin I am not able to easily obtain that here.  I don't know where to get the UserID from (e.g. from the JWT Token?)